ods_ci/tests/Resources/Common.robot

```diff
@@ -305,3 +305,38 @@ Extract URLs From Text
     [Arguments]    ${text}
     ${urls}=     Get Regexp Matches   ${text}   (?:(?:(?:ftp|http)[s]*:\/\/|www\.)[^\.]+\.[^ \n]+)
     RETURN    ${urls}
+
+Run And Watch Command
+  [Documentation]    Run any shell command (including args) with optional:
+  ...    Timeout in minutes: 10 by default.
+  ...    Expected text: Verify the text exists in command output.
+  [Arguments]    ${command}    ${timeout_min}=10    ${expected_text}=${NONE}
+  Log    Watching command output: ${command}   console=True
+  @{args} =    Split String    ${command}
+  ${process_log} =    Set Variable    ${OUTPUT DIR}/${args}[0].log
+  Create File    ${process_log}
+  Create File    ${process_log}.old
```

Comment:
what is the idea behind old and new log file?

Reply:
Good question: We need to direct stdout and stderr to external log (new file), but each few seconds track the new lines appended to the file. For that I added old file that is a copy of new file - before new data was appended to the new file (that is constantly getting proccess output). Each interval I compare between old file and new file data, and only print the diff into console in real time.

Reply:
I think it better you directly monitor the process output and log to console instead of open file and write to it.That would be easy path to go and also save some time might be in second

Reply:
You don't need multiple file ..you can monitor the out just via alias and print the out put
` alias= prep
  ${result}   Wait For Process   prep   timeout=10 secs
  ${status}   Is Process Running   prep`

Reply:
I'm not sure I'm following your suggestion how to compare stdout of external process, without using a reference to the stdout before the new stdout was sent to. 
Anyway, I have changed the logic to use a temporary file for the old stdout/stderr, and now keep just the latest process log in artifacts.